N/normaliz/build_tarballs.jl

```diff
@@ -71,8 +71,9 @@ products = [
 dependencies = [
     Dependency("GMP_jll", v"6.2.0"),
     Dependency("MPFR_jll", v"4.1.1"),
-    Dependency("FLINT_jll"; compat = "~300.100.300"),
-    Dependency("nauty_jll"; compat = "~2.6.13"),
+    Dependency("OpenBLAS32_jll", v"0.3.28"),
+    Dependency("FLINT_jll"; compat = "~300.200.0"),
+    Dependency("nauty_jll"; compat = "~2.8.9"),
```

Comment:
```suggestion
    Dependency("nauty_jll"; compat = "~2.8.10"),
```
Bump nauty again to fix aarch64-freebsd